ip/src/main/java/BuddyBot.java

```diff
@@ -0,0 +1,94 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class BuddyBot {
```

Comment:
Comments are missing for most methods and classes. Please add them as they increase the readability of the code.